specification/translation/Azure.AI.DocumentTranslation/client.tsp

```diff
@@ -41,49 +41,131 @@ interface SingleDocumentTranslationClient {
   documentTranslate is DocumentTranslation.DocumentTranslationOperations.documentTranslate;
 }
 
-// model overrides
-@@access(DocumentTranslation.DocumentTranslateResult,
+@@clientName(DocumentTranslation.DocumentTranslateBody.body,
+  "documentTranslateContent",
+  "csharp"
+);
+
+@@clientName(DocumentTranslation.DocumentTranslateBody.body,
+  "documentTranslateContent",
+  "java"
+);
+
+@@clientName(DocumentTranslation.StartTranslationBody.body,
+  "startTranslationDetails",
+  "csharp"
+);
+
+@@clientName(DocumentTranslation.StartTranslationBody.body,
+  "startTranslationDetails",
+  "java"
+);
+
+@@access(DocumentTranslationClient.getSupportedFormats,
   Access.internal,
   "python"
 );
-@@access(DocumentTranslation.FileFormatType, Access.internal, "python");
-@@access(DocumentTranslation.StorageSource, Access.internal, "python");
-@@access(DocumentTranslation.AdditionalOutputFileFormat,
-  Access.internal,
+@@access(DocumentTranslation.FileFormat, Access.public, "python");
```

Comment:
Why do you need to specify this? Is it not getting generated by default?

Reply:
When setting the getSupportedFormats method to internal, all associated models are not generated. the access/usage lines are just adding them back so we can use the models